aiqleads/core/project_tracking.py

```
@@ -8,15 +8,21 @@
 import logging
```

Ruff (0.8.2)

`logging` imported but unused

Remove unused import: `logging`

(F401)